py/desisim/io.py

```diff
@@ -917,8 +917,11 @@ def empty_metatable(nmodel=1, objtype='ELG', subtype='', add_SNeIa=None):
                            data=np.zeros(nmodel)))
     meta.add_column(Column(name='MAG', length=nmodel, dtype='f4',
                            data=np.zeros(nmodel)-1))
-    meta.add_column(Column(name='DECAM_FLUX', shape=(6,), length=nmodel, dtype='f4'))
-    meta.add_column(Column(name='WISE_FLUX', shape=(2,), length=nmodel, dtype='f4'))
+    meta.add_column(Column(name='FLUX_G', length=nmodel, dtype='f4'))
+    meta.add_column(Column(name='FLUX_R', length=nmodel, dtype='f4'))
+    meta.add_column(Column(name='FLUX_Z', length=nmodel, dtype='f4'))
+    meta.add_column(Column(name='FLUX_W1', length=nmodel, dtype='f4'))
+    meta.add_column(Column(name='FLUX_W2', length=nmodel, dtype='f4'))
```

Comment:
You include unit metadata for the other columns in this table; let's do it for these new columns too.